### PR TITLE
Member views books

### PR DESCRIPTION
**Pull request by DaoDucDanh**
## Related Tickets
- [#48540](https://edu-redmine.sun-asterisk.vn/issues/48540)

## WHAT (optional)
- User can view a list of book for lending.

## HOW
- I Added a index book view for user. 

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by DaoDucDanh** [comment]
ready

**Comment by DaoDucDanh** [comment]
ready

File: app/helpers/books_helper.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module BooksHelper
+  def to_status_string book
```
Comment (ngocvt-0484) [inline app/helpers/books_helper.rb]:
```suggestion
  def book_status_i18n book
```

File: config/routes.rb [hunk 2]
```diff
@@ -10,6 +10,8 @@
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
 
+    get "books", to: "books#index"
```
Comment (ngocvt-0484) [inline config/routes.rb]: dùng resource only index đi e

**Comment by DaoDucDanh** [comment]
ready

**Pull request merged**